Code/GraphMol/RascalMCES/mces_catch.cpp

```diff
@@ -542,7 +542,7 @@ TEST_CASE("single fragment") {
     res = rascalMCES(*m1, *m2, opts);
     REQUIRE(res.front().getNumFrags() == 1);
     REQUIRE(res.front().getBondMatches().size() == std::get<3>(test));
-    REQUIRE(res.front().getLargestFragSize() ==
+    REQUIRE(static_cast<unsigned>(res.front().getLargestFragSize()) ==
```

Comment:
getLargestFragSize() should only return a positive integer or 0 - it's the number of atoms in a molecule fragment.  However, to support lazy evaluation, the member variable in RascalResult is an int initialised to -1.  What's the best way of handling that?  getLargestFragSize() could return an unsigned int and then this cast will be unnecessary.

Reply:
I think we can make `getLargestFragSize()` return `unsigned int`. It makes sense, since it returns a size. I can take care of that.